### ISSUE TITLE
[ISSUE] Alarm popup window opens on multiple devices simultaneously, stays open after press

### ISSUE THREAD

**Issue by sharkpunch5** [jimz011/homekit-infused#65]
**Description**

_Part 1_ - When I arm/disarm the alarm from my home screen in my iphone, the 'alarm options' popups simultaneously appears on my desktop (chrome browser), is it possible to limit the appearance of this popup to the device it was activated on?

_Part 2_ - after clicking an option the alarm options popup remains open, could the default behaviour be to close this window and return the the home screen? (this approach may indirectly resolve the issue above for most situations)

_Part 3_ - Feature request - is it possible to provide an option to remove the 'are you sure' prompt for the alarm popup?

![image](https://user-images.githubusercontent.com/28176100/102480326-399fd880-40b4-11eb-849c-1565a4e03233.png)

**HA Version:** 2020.12.1

**Comment by EliotHertenstein** [comment]
The same issue regarding multiple devices for the lights button popup. They might not be using device-id in Browser_Mod

**Comment by jimz011** [comment]
Correct, I have removed it so that browser_mod could be updated to the latest version. Since browser_mod changed the way it uses device_id: this it isn't/wasn't possible with that version of browser-mod. It simply didn't/doesn't support custom-cards in it (which I have used for those popups).

Maybe it has been corrected now, but I did not had time to look into it. I also mentioned this in the notes/breaking changes why I removed it and what would happen for the time being. I hope this will only be temporary, though I thought updating to the latest version is probably worth the temporary fix.

I need to do the same for layout-card which I am using a very old version of. I will try to address this for the next release (which is v4). I am already fixing a lot of stuff that needs fixing (like the subtitle not showing the same size when no notification is active, the navigation bar, which I recently released and a lot of other stuff like auto-filling which is now an addon and will become core HKI in v4).

So consider this as something that is on the priority list to fix for v4
